cmd/fyne_demo/data/icons.go

```diff
@@ -11,7 +12,7 @@ type ThemedResource struct {
 }
 
 func isLight() bool {
-	r, g, b, _ := fyne.CurrentApp().Settings().Theme().TextColor().RGBA()
+	r, g, b, _ := theme.TextColor().RGBA()
```

Comment:
If Text and Icon use the same colour, perhaps it should be called `Foreground` colour (consistent with `Background` colour)?

Reply:
thanks yes, done